src/api.ts

```diff
@@ -142,3 +142,90 @@ export async function createAccessPass(
   );
   return response.data;
 }
+
+export type FieldValue = boolean | string | number;
+
+export interface TestResults {
+  details: { [key: string]: FieldValue }; // TODO: proper typing based on test type
```

Comment:
Because json schema can support any type, any sort of strict typing is a bit futile...